.github/workflows/pullpreview.yml

```diff
@@ -17,6 +17,14 @@ on:
         description: The name of an Artifact which will be extracted while preserving file paths.
         required: false
         type: string
+      DNS:
```

Comment:
```suggestion
      CUSTOM_DOMAIN:
```

Reply:
or `DOMAIN`

Reply:
Changed it to `DOMAIN` like suggested. Done.